cfme/fixtures/service_fixtures.py

```diff
@@ -159,12 +160,15 @@ def create_catalog_item(appliance, provider, provisioning, dialog, catalog,
 
 @pytest.fixture
 def order_service(appliance, provider, provisioning, dialog, catalog, request):
+    # BZ 1646333 - Delete this request button is not shown in service Request details page
+    # The above BZ got closed because of  INSUFFICIENT_DATA.
     """ Orders service once the catalog item is created"""
-
     if hasattr(request, 'param'):
-        param = request.param
+        vm_count = request.param['vm_count'] if 'vm_count' in request.param else '1'
+        console_test = request.param['console_test'] if 'console_test' in request.param else False
```

Comment:
Can you please elaborate why `console_test` is defaulting to `False` over `None`?

Reply:
The existing code already specifies that the default value of the 'console_test' keyword argument is False.
Am I missing something ?

```suggestion
        def create_catalog_item(appliance, provider, provisioning, dialog, catalog,
        console_test=False):
```

Reply:
Ah right, I see it now. Thanks for pointing out.